components/shared/publizonReader/helper.ts

```diff
@@ -39,3 +39,15 @@ export const appendAsset = ({ src, type }: AssetType) => {
     document.head.appendChild(linkElement)
   }
 }
+
+export const removeAsset = ({ src, type }: AssetType) => {
```

Comment:
I guess `AssetType`should be: `TAssetType` in order to follow our naming convention